### PR TITLE
Clarifying about geo

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2694

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/allow-syncing-only-on-specific-domains.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ To make sure that users sync OneDrive files only on managed computers, you can c
  
 > [!NOTE]
 > These settings apply to SharePoint sites as well as OneDrive.
+> In a Multi-Geo environment, this setting can be configured separately for each geo location and applies to users with that perferred data location.
```
Comment (JohanFreelancer9) [inline OneDrive/allow-syncing-only-on-specific-domains.md:32]:
```suggestion
> In a multi-geo environment, this setting can be configured separately for each geo location and applies to users with that preferred data location.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**